lib/components/viewers/live-stop-times.tsx

```diff
@@ -124,6 +128,26 @@ class LiveStopTimes extends Component<Props, State> {
       this._startAutoRefresh()
   }
 
+  renderDay = (homeTimezone: string, day: number, now: Date): JSX.Element => {
+    const formattedDay = utcToZonedTime(day * 1000, homeTimezone)
+    return (
+      <React.Fragment key={day}>
+        {/* If the service day is not today, add a label */}
```

Comment:
A very small thing... inserting a label visually "breaks" the grouping (the edges of the arrivals it sit between are not rounded).

Reply:
This is actually intentional! The idea was for the entire departure container to be rounded, so the middle rows should stay square. Original wireframe for reference: 

![image](https://user-images.githubusercontent.com/115499534/227360197-39ab89ad-3d53-459b-917c-837240c24dd7.png)